static/app/views/performance/newTraceDetails/traceTree.tsx

```diff
@@ -1136,6 +1128,18 @@ export class TraceTreeNode<T extends TraceTree.NodeValue> {
     return node;
   }
 
+  addErrorsFromArray(errors: TraceErrorType[]) {
+    for (const error of errors) {
+      this.errors.add(error);
+    }
+  }
+
+  addPerformanceIssuesFromArray(performanceIssues: TracePerformanceIssue[]) {
+    for (const issue of performanceIssues) {
+      this.performance_issues.add(issue);
+    }
+  }
+
```

Comment:
Wrappers around for loops are too low value to exist. It is better to be explicit and drop these